marketplace/core/types/views.py

```diff
@@ -30,4 +31,5 @@ def perform_destroy(self, instance):
         channel_uuid = instance.config.get("channelUuid")
 
         if channel_uuid is not None and settings.USE_GRPC:
```

Comment:
The name of this `USE_GRPC` variable no longer makes sense, but that doesn't need to be changed now. I will add this to the backlog.